packages/next/src/build/webpack/plugins/nextjs-require-cache-hot-reloader.ts

```diff
@@ -66,6 +47,29 @@ export function deleteCache(filePath: string) {
   return false
 }
 
+export function deleteAppClientCache() {
```

Comment:
Moving this down makes the diff a bit unnecessarily complicated 🙃

Reply:
yeah. eslint forced me to do that... declare before use... :-|